### DISCUSSION TITLE
Question: Don't move cursor until I select something

### DISCUSSION THREAD

**Discussion by hqkhan** [ibhagwan/fzf-lua#1141] [General]
When I'm live grepping around, the cursor of my buffer (NOT fzf-lua's window cursor) moves to the line number of my current selection even if the selection is from different file altogether. 

Is there a way for my buffer's cursor not to move until I've made a selection? Sometimes, I'm looking through the selections and decide to cancel the search but my cursor has moved me somewhere entirely different.

**Comment by ibhagwan** [comment]
Not sure I understand what's happening, why would your cursor move?

Unless I pressed enter on a selection my cursor never moves, either I'm misunderstanding this or it's a bug.

**Reply by hqkhan** [reply]
:o wait... I swear it was happening in my session. I re-launched neovim and now it's behaving as expected...I had my session on long enough to think that it was a bug...Everything looks fine now...

**Reply by ibhagwan** [reply]
Hard for me to imagine which code of fzf-lua will be bugged to trigger a cursor move without accepting selection but who knows, stranger things have happened or potentially a bug somewhere other than fzf-lua :-)

**Reply by hqkhan** [reply]
Funnily enough, it happened to me a couple more times but I can't repro it yet. I'll try to get a repro and be back lmao

**Reply by ibhagwan** [reply]
Np, keep me posted :)

**Reply by hqkhan** [reply]
Ok, this was a pain but I finally figured out the issue. TLDR; not an fzf-lua issue...

I had:

```
vim.wo[0].cursorbind = true
vim.schedule(function()
  vim.cmd.wincmd('p')
  vim.wo[0].cursorbind = true
  vim.cmd.wincmd('p')
end)
```

in `fugitiveblame.lua` so cursor shows up for this buffer and binded it together so they move together...

Now that we're on the topic, how do you use `git blame` option? I use it quite a bit to see who wrote what. I'm assuming this will need an aucmd where it implements the above and un-does it on leaving...

**Reply by ibhagwan** [reply]
> Now that we're on the topic, how do you use git blame option? I use it quite a bit to see who wrote what. I'm assuming this will need an aucmd where it implements the above and un-does it on leaving...

 I have a toggle to enable git blame virtual text in git signs or a pop up for git blame for the specific line I’m on
> Line blame popup
![IMG_2048](https://github.com/ibhagwan/fzf-lua/assets/59988195/765771a7-a617-40b7-a60c-bd45e873c283)

> Line blame virtual text
![IMG_2049](https://github.com/ibhagwan/fzf-lua/assets/59988195/c2142faf-6247-4c3a-98e0-bfe18fdeed52)

**Reply by hqkhan** [reply]
I see. That's interesting although I much prefer a longer version of git blame for all lines on my screen. Either way, I'm glad this was resolved cuz it was bothering me a lot lol.
